mob.go

```diff
@@ -942,7 +949,7 @@ func startTimer(timerInMinutes string, configuration Configuration) {
 
 	if startRemoteTimer {
 		timerUser := getUserForMobTimer(configuration.TimerUser)
-		err := httpPutTimer(timeoutInMinutes, room, timerUser, configuration.TimerUrl)
+		err := httpPutTimer(timeoutInMinutes, room, timerUser, configuration)
```

Comment:
I don't want to pass in the configuration to the httpPutTimer method. It's a preparation for extracing this without the configuration to a separate package.